### DISCUSSION TITLE
Persistent forced sign-out [Jan 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4967] [Discord Threads]
> 🙋‍♂️ **CynR**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### That's weird behaviour. We have not encountered something like that so far. If issue persists, help us with relevant details on Github issue .    
#### https://github.com/nocodb/nocodb/issues  
#  
> 🙋‍♂️ **CynR**  
#### thanks for the feedback, not sure if it is something I have done wrong (newbie), or a bug but my instance is not usable, kicks me to signup page every few minutes - seems to think I am a guest from the alerts.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1055444609045635092)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
